src/components/CommentList/CommentList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './CommentList.scss';
+
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+type Props = {
+  comments: PostComment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <>
+    <p className="subtitle">Comments:</p>
+    {comments.map(comment => (
+      <ul className="content" key={comment.id}>
```

Comment:
check the structure you have now, every comment should be a `li`, not `ul`
![image](https://user-images.githubusercontent.com/64437769/150115692-f2b9d3c7-ddf0-4b68-8e6f-1c5e43a8163c.png)